src/main/java/commands/CreateTask.java

```diff
@@ -0,0 +1,42 @@
+package commands;
+
+import myTasks.Deadline;
+import myTasks.Event;
+import myTasks.Task;
+import myTasks.Todo;
+
+import java.util.List;
+
+/**
+ * The CreateTask class contain methods to create new tasks for the user to track.
+ */
+public class CreateTask {
+    /**
+     * The method todo creates a task without restrictions.
+     * @param description contains the description of the task.
+     * @param list contains the list of tasks that are currently being tracked.
+     */
+    public static void todo(String description, List<Task> list) {
+        list.add(new Todo(description));
+    }
+
+    /**
+     * The method deadline creates a task that must be completed before a certain time.
+     * @param description contains the description of the task.
+     * @param list contains the list of tasks that are currently being tracked.
+     */
+    public static void deadline(String description, List<Task> list) {
+        String[] restOfPara  = description.split("/by ", 2);
```

Comment:
Not sure if I like the variable name because Para can be inferred as paragraph, parameter and etc. Might be better if you just spell out the whole word for readability.